src/main/java/md/miller1995/Dealership/models/entities/UserEntity.java

```diff
@@ -0,0 +1,26 @@
+package md.miller1995.Dealership.models.entities;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDate;
+
+@Entity
+@Table(name = "users_auth")
```

Comment:
The most important annotations should be at the bottom of the list. Annotations for mapping the entity are more important than Lombok's annotations.
Also, we have `UserEntity` but the table name is `users_auth`? Do we have more User entities? The naming between the class and the table should be the same, except the fact that the table should use the plural.